Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugins/Plugins/plugins.json

```diff
@@ -30,14 +25,9 @@
             "text": "In the context of a community-driven project, a plugin is a device that serves to distribute community-generated intelligence. The project distributes several plugins contributed by the community.",
             "translations": [
                 {
-                    "language": "EL",
-                    "text": "Στο πλαίσιο ενός έργου που βασίζεται σε κοινότητα, ένα πρόσθετο είναι μια συσκευή που χρησιμεύει για τη διανομή πληροφοριών που δημιουργείται από την κοινότητα. Το έργο διανέμει πολλά πρόσθετα που συνεισφέρει η κοινότητα.",
-                    "updated": 1641828813002
-                },
-                {
-                    "language": "DE",
-                    "text": "Im Kontext eines community-gesteuerten Projekts ist ein Plugin ein Gerät, das dazu dient, von der Community generierte Intelligenz zu verbreiten. Das Projekt vertreibt mehrere Plugins, die von der Community beigesteuert werden.",
-                    "updated": 1641764929240
+                    "language": "TR",
+                    "text": "Topluluk odaklı bir proje olarak, eklentiler, topluluk tarafından oluşturulan zekayı dağıtmaya hizmet eden bir cihazdır. Proje, topluluk tarafından sağlanan çeşitli eklentileri dağıtır.",
+                    "updated": 1644582550980
                 }
```

Comment:
> You are accidentally overwriting two translations. Do you know how to fix them?

There should not be such a mistake, because I am sure that I did everything right. but i don't know how to fix it..